app/controllers/nonsipp/employercontributions/EmployerContributionsCYAController.scala

```diff
@@ -80,7 +80,7 @@ class EmployerContributionsCYAController @Inject()(
         for {
           userAnswers <- EitherT(userAnswersWithSectionCompleted.pure[Future])
           updatedAnswers <- Future
-            .fromTry(userAnswers.set(EmployerContributionsSectionStatus(srn), SectionStatus.InProgress))
+            .fromTry(userAnswers)
```

Comment:
is this line doing anything anymore?

Reply:
No, but I had some issues when I tried to remove it, so I left it.